pages/characters.js

```diff
@@ -1,11 +1,15 @@
 import { Title } from '../components/Header';
 import ShowCharacter from '../components/ShowCharacter';
+import Link from 'next/link';
 
 export default function Characters() {
   return (
     <>
       <Title>My Characters</Title>
       <ShowCharacter />
+      <Link passHref href="/createCharacter">
+        <button as="a">Back to Create Character</button>
+      </Link>
```

Comment:
A `button` is not for linking and you are making it to an `a`-tag with the `as="a"`. So just use the `a`-tag for this.